boms/user/server/ee/standard/with-tools/pom.xml

```diff
@@ -0,0 +1,92 @@
+        <dependencies>
+            <dependency>
+                <groupId>${bomGroupId}</groupId>
+                <artifactId>${bomArtifactIdPrefix}-tools</artifactId>
+                <version>${project.version}</version>
+                <type>pom</type>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
+
+    <dependencies>
+        <!-- depende on tools BOM builder to ensure such BOM is built before this builder runs -->
+        <dependency>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>wildfly-tools-builder</artifactId>
+            <version>${project.version}</version>
+            <type>pom</type>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.wildfly.plugins</groupId>
+                <artifactId>wildfly-bom-builder-plugin</artifactId>
+                <inherited>false</inherited>
+                <executions>
+                    <execution>
+                        <id>build-bom</id>
+                        <goals>
+                            <goal>build-bom</goal>
+                        </goals>
+                        <configuration>
+                            <parent>
+                                <!-- parent is the BOM built by parent builder -->
+                                <groupId>${bomGroupId}</groupId>
+                                <artifactId>${bomArtifactIdPrefix}-ee</artifactId>
+                                <version>${project.version}</version>
+                                <relativePath/>
+                            </parent>
+                            <bomGroupId>${bomGroupId}</bomGroupId>
+                            <bomArtifactId>${bomArtifactIdPrefix}-ee-with-tools</bomArtifactId>
+                            <bomVersion>${project.version}</bomVersion>
+                            <bomName>${full.dist.product.release.name} BOMs: EE With Tools</bomName>
+                            <bomDescription>${full.dist.product.release.name}: EE Dependency Management, with useful tools</bomDescription>
+                            <!-- ensure no deps are re-included from parent -->
+                            <includeDependencies combine.self="override"/>
+                            <!-- the only entry in the BOM's dep mngmnt should be an import for the tools BOM -->
+                            <importDependencies>
+                                <dependency>
+                                    <groupId>${bomGroupId}</groupId>
+                                    <artifactId>${bomArtifactIdPrefix}-tools</artifactId>
+                                    <type>pom</type>
+                                </dependency>
+                            </importDependencies>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+        </plugins>
+    </build>
```

Comment:
@emmartins This bit from the current ee-with-tools/target/bom-pom.xml  is missing from the result from this build:

```
  <build>
    <pluginManagement>
      <plugins>
        <plugin>
          <groupId>org.wildfly.plugins</groupId>
          <artifactId>wildfly-maven-plugin</artifactId>
          <version>5.0.1.Final</version>
        </plugin>
      </plugins>
    </pluginManagement>
  </build>
```

Reply:
fixed